src/tabs/RP_Getting_Started.tab

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomTab xmlns="http://soap.sforce.com/2006/04/metadata">
```

Comment:
@davidhabib @szeitz my only concern here is that tabs are tied to the VF page and cannot be updated. after created. What would be the alternative here?